tests/transi/transi_test.h

```diff
@@ -35,6 +35,7 @@
 #define ASSERT( assertion ) do {\
   if( !(assertion) ) {\
     printf("ERROR: Assertion `%s' failed @%s:%d\n",#assertion,__FILE__,__LINE__);\
+    TRANS_CHECK( trans_delete(&trans) );\
```

Comment:
Is this required? we're exiting anyway?

Reply:
It's more of a neatness thing so that GPU arrays are properly deallocated. Without this you get a cryptic error message (which I can't remember off the top of my head) on exit when running on AMD devices with the Cray stack. Nvidia GPUs don't show the same behaviour so it may be that the NVHPC-compiled version has some kind of device garbage collection.